beginner_source/template_tutorial.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+
+"""
+Template Tutorial
+=================
+
+**Author:** `FirstName LastName <https://github.com/username>`_
+
+.. grid:: 2
+
+   .. grid-item-card:: :octicon:`mortar-board;1em;` What you will learn
+
+      * Item 1
+      * Item 2
+      * Item 3
+
+   .. grid-item-card:: :octicon:`list-unordered;1em;` Prerequisites
+
+      * PyTorch v2.0.0
+      * GPU ???
+      * Other items 3
+
+If you have a video, add it here like this:
+
+.. raw:: html
+
+   <div style="margin-top:10px; margin-bottom:10px;">
+     <iframe width="560" height="315" src="https://www.youtube.com/embed/IC0_FRiX-sw" frameborder="0" allow="accelerometer; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+   </div>
+
+To test your tutorial locally, make a copy of this repo and add only your
+tutorial removing everything. Then run ``make html``. Verify that all outputs were generated correctly
+in the created HTML.
```

Comment:
Do we want to start requiring that people run **pylint** or **flake8** or **black**? Let's discuss--but not a reason to wait on merging.